Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2856,6 +2856,21 @@ extension BrowserViewController: ReaderModeBarViewDelegate {
 }
 
 extension BrowserViewController: ContextMenuHelperDelegate {
+    @objc
+    private func image(image: UIImage, didFinishSavingwithError error: NSError?, contextInfo: UnsafeRawPointer?) {
```

Comment:
Just to be sure, this code is for iOS 12 only right? Since `ContextMenuHelper` is a iOS 12 helper

Reply:
Yes this should only be for iOS 12. This callback doesn't get triggered on 13.